File: Backend/src/main/java/com/example/backendsitemate/Entity/Issue.java
package com.example.backendsitemate.Entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "sitemate")
@AllArgsConstructor
@NoArgsConstructor
public class Issue {

    @Id
    @Setter
    @Getter
    private String _id;

    @Setter
    @Getter
    private String title;

    @Setter
    @Getter
    private String description;


    @Override
    public String toString(){
        return "Sitemate Issue{" +
                "_id='" + _id + '\'' +
                ", Title='" + title + '\'' +
                ", Description='" + description +
                '}';
    }
}

File: Frontend/sitemateFrontend/src/app/Issue.ts
export class Issue {
    _id: string;
    title: string;
    description : string;

    constructor(_id:string,title:string,description:string)
    {
        this._id = _id;
        this.description = description;
        this.title = title;
    }

    getId()
    {
        return this._id;
    }

    getDescription()
    {
        return this.description;
    }

    getTitle()
    {
        return this.title;
    }
    
   }
